services/wallet/datastore.go

```diff
@@ -616,17 +614,29 @@ func (pg *Postgres) LinkWallet(ctx context.Context, id string, userDepositDestin
 	return nil
 }
 
-// CustodianLink - representation of wallet_custodian record
+// TODO(clD11): CustodianLink represent a wallet_custodian. Review during wallet refactor
+
+// CustodianLink representation a wallet_custodian record.
 type CustodianLink struct {
-	WalletID           *uuid.UUID `json:"wallet_id" db:"wallet_id" valid:"uuidv4"`
-	Custodian          string     `json:"custodian" db:"custodian" valid:"in(uphold,brave,gemini,bitflyer)"`
-	CreatedAt          time.Time  `json:"created_at" db:"created_at" valid:"-"`
-	UpdatedAt          *time.Time `json:"updated_at" db:"updated_at" valid:"-"`
-	LinkedAt           time.Time  `json:"linked_at" db:"linked_at" valid:"-"`
-	DisconnectedAt     *time.Time `json:"disconnected_at" db:"disconnected_at" valid:"-"`
-	DepositDestination string     `json:"deposit_destination" db:"deposit_destination" valid:"-"`
```

Comment:
It took me a while to notice this was gone. Is it a good idea to include this with the Solana feature? Or should such a small change go separately? Maybe a simple small PR would suffice?

Reply:
None of this is used and shouldnt be missed